css/values.css

```diff
@@ -0,0 +1,557 @@
+Page Component Styles (icon grid, team member card)
+===================================
+*/
+.hero {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.hero img {
+	width: 100%;
+}
+.values-card {
+	min-height: 204px;
+	color: var(--color-white);
+	border: none;
+	border-radius: 10px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin: 0px 20px;
+}
+.values-card img {
+	margin-bottom: 20px;
+}
+.values-card h2 {
+	margin: 20px 0px 0px 0px;
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+}
+.values-card p {
+	display: flex;
+	text-align: left;
+	font-size: var(--font16);
+	font-weight: var(--font-weight-regular);
+	line-height: var(--line-height20);
+	margin: var(--spacing20);
+}
+/* Turns Our Values Title Black */
+.values-head {
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+	margin-bottom: 100px;
+}
+
+.h1-black {
+	color: var(--color-onyx);
+	margin: 60px 0 20px 0;
+}
+
+.values-head p {
+	color: var(--color-onyx);
+	margin: 0 20px 0;
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.copperback {
+	background-color: var(--color-copper);
+	display: flex;
+	flex-direction: column;
+	text-align: center;
+}
+
+/* Adds a background color in lieu of those rectangle shapes */
+
+.onyxback {
+	background-color: var(--color-onyx);
+}
+
+/* Adds a background color in lieu of those rectangle shapes */
+
+.amaraback {
+	background-color: var(--color-amaranth);
+}
+
+.sustainability-text,
+.authenticity-text,
+.education-text,
+.tech-text {
+	display: flex;
+	flex-direction: column;
+	border-radius: 10px;
+}
+
+.copperback,
+.sustainability-text,
+.grapeback,
+.authenticity-text,
+.education-text,
+.tech-text {
+	margin-bottom: 20px;
+}
+
+.sustainability-text {
+	background-color: var(--color-onyx);
+}
+.authenticity-text {
+	background-color: var(--color-amaranth);
+}
+.education-text {
+	background-color: var(--color-yaleblue);
+}
+.tech-text {
+	background-color: var(--color-teal);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.grapeback {
+	background-color: var(--color-cybergrape);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.blueback {
+	background-color: var(--color-yaleblue);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.tealback {
+	background-color: var(--color-teal);
+}
+/* Adds a background color in lieu of those rectangle shapes */
+.support {
+	background-color: var(--color-yaleblue);
+	margin-top: 90px;
+}
+.support-header,
+.support-text {
+	margin: 0px 0px 0px 20px;
+}
+.support-header {
+	display: flex;
+	flex-direction: row;
+	justify-content: left;
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+}
+/* Turns We Support to Red */
+/* inline block allows the alignment with heart icon but doesn't fill desktop */
+.support-header h4,
```

Comment:
Please double-check font sizes and spacing for this section, it looks a bit off. 
![Uploading Screen Shot 2024-01-19 at 11.15.00 AM.png…]()